src/Components/SearchPage/SearchPage.js

```diff
@@ -3,14 +3,23 @@ import "./SearchPage.scss";
 import SearchBar from "../SearchBar/SearchBar";
 import Logo from "../assets/DiscoverItLogo.svg";
 import NavBar from "../NavBar/NavBar";
+import { Link } from "react-router-dom";
 
 const SearchPage = ({ city, setCity }) => {
+  const exploreCity =
```

Comment:
Since you're only needing to return a value if the assertion is truthy, instead of a ternary I would just use a single if statement here.